spec/sparql.bnf

```diff
@@ -0,0 +1,284 @@
+QueryUnit                 ::= Query 
+Query                     ::= Prologue<br/>( SelectQuery | ConstructQuery | DescribeQuery | AskQuery )<br/>ValuesClause
```

Comment:
```suggestion
Query ::= Prologue
          ( SelectQuery | ConstructQuery | DescribeQuery | AskQuery )<br/>ValuesClause
```

Reply:
@gkellogg - as you know , the BNF is generated, not hand edited. I'll make these changes to the script that generates the BNF text otherwise they will get lost.